content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -146,7 +146,7 @@ different Kubernetes components.
 | `NodeDisruptionExclusion` | `true` | Beta | 1.19 | 1.20 |
 | `NonPreemptingPriority` | `false` | Alpha | 1.15 | 1.18 |
 | `NonPreemptingPriority` | `true` | Beta | 1.19 | |
-| `PodDeletionCost` | `false` | Alpha | 1.21 | |
+| `PodDeletionCost` | `true` | Beta | 1.21 | 1.22 |
```

Comment:
I think you mean
```suggestion
| `PodDeletionCost` | `true` | Beta | 1.22 |   |
```

Reply:
oops, I clearly didn't look at the header :)
done.